src/Pages/PassengerInfo/PassengerInfo.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import Accordian from './Accordian';
+import Accordian2 from './Accordian2';
+import Accordian3 from './Accordian3';
+
+import styled from 'styled-components';
```

Comment:
```suggestion
import React, { useState } from 'react';
import styled from 'styled-components';
import Accordian from './Accordian';
```
가독성을 위해 import 순서 맞춰주세요!
- React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)